app/controllers/redirection_controller.rb

```diff
@@ -16,6 +16,22 @@ def redirect_latest
     redirect_to(finder_path("search/all", params: { order: "updated-newest" }.merge(redirect_params)))
   end
 
+  def redirect_consultations
+    topics = {}
+    topics["level_one_taxon"] = params[:topics] if params[:topics]
+    topics["level_two_taxon"] = params[:subtaxons] if params[:subtaxons]
+
+    redirect_params = params.merge(topics)
+                            .slice(:departments, :level_one_taxon, :level_two_taxon, :world_locations, :content_store_document_type)
+                            .permit(:level_one_taxon, :level_two_taxon, departments: [],  world_locations: [], content_store_document_type: [])
```

Comment:
There's two spaces between the end of departments array and world locations.  It's fixed in the next commit, but good to do that here too.

Reply:
👍